Tests/LinuxMain.swift

```diff
@@ -0,0 +1,8 @@
+import XCTest
+
+import BotTests
+
+var tests = [XCTestCaseEntry]()
+tests += BotTests.__allTests()
```

Comment:
I'm guessing this was auto-generated by `swift test --generate-linuxmain`? (otherwise, not sure why we use a `var` then `+=` instead of affecting `let tests = BotTests.__allTests()` direclty... and it's sad that they don't insert a comment telling that the file was auto-generated... but I guess you can't do much about it)

Reply:
It was automatically generated, I haven't edited this file.

Reply:
Interesting, `Tests/BotTests/XCTestManifests.swift` includes a comment with Xcode 10.2.1 but nothing has changed in `Tests/LinuxMain.swift`.
<img width="1004" alt="image" src="https://user-images.githubusercontent.com/1098602/58912455-f8ceb180-8711-11e9-960e-dbd1dfc42eba.png">